sync/src/tasks/block_sync_task.rs

```diff
@@ -402,16 +397,10 @@ where
         &self,
         mut block_headers: Vec<BlockHeader>,
     ) -> Result<Vec<Block>> {
-        // let mut ancestors = vec![];
-        // let mut absent_block_headers = vec![];
         let mut absent_blocks_map = HashMap::new();
         loop {
             let mut absent_blocks = vec![];
-            self.find_absent_parent_dag_blocks_for_blocks(
-                block_headers,
-                // &mut ancestors,
-                &mut absent_blocks,
-            )?;
+            self.find_absent_parent_dag_blocks_for_blocks(block_headers, &mut absent_blocks)?;
             if absent_blocks.is_empty() {
                 return Ok(absent_blocks_map.into_values().collect());
             }
```

Comment:
This function fetch_block return Result<Vec<(HashValue, Block)>>. It seems that you don't use HashValue. Why don't you return Result<Vec<Block>>. Maybe fetch_blocks is better, just from previous commit log.

Reply:
Okay, I will add an issue and fix it next version.